python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1329,6 +1329,9 @@ keys:
                     - "mpls"
                     - "path-selection"
                   description: Transport encapsulation for the neighbor.
+                next_hop_self_source_interface:
+                  type: str
+                  description: Source interface name for MPLS encapsulation. Requires `encapsulation` to be set as `mpls`.
```

Comment:
Can you add the same comment line 1261 for the same key under neighbor_default for consistency? (I cannot comment up there in Github)

Reply:
added